src/renderer/src/components/trigger/TriggersSlice.ts

```
@@ -1,3 +1,5 @@
+import { useSelector } from 'react-redux';
```

ESLint

[error] 'useSelector' is defined but never used.

(@typescript-eslint/no-unused-vars)